queries/luau/highlights.scm

```diff
@@ -43,19 +43,34 @@
     "end"
   ] @keyword.conditional)
 
+(if_expression
+  [
+    "if"
+    "then"
+  ] @keyword.conditional)
+
 (elseif_statement
```

Comment:
Is that correct, or should it be `elseif_expression`? (The mix of  `expression` and `statement` is weird.)

Reply:
Yeah that's correct - the expression is for luau's shorthand if expressions like so
```luau
local a = if foo then 1 elseif bar then 2 else 3
```

Reply:
So some nodes derive from Lua and other ones are new in this grammar, and it's pretty much arbitrary ;)